### PR TITLE
Rename master to apiserver in test/integration

### PR DESCRIPTION
**Pull request by mengjiao-liu**
#### What type of PR is this?
/kind cleanup
/wg naming
/area test

#### What this PR does / why we need it:

xref #94407
As https://github.com/kubernetes/kubernetes/pull/100181#issuecomment-805017091 commented, this PR change codes less than 50 lines at a time. It would be friendly to review.
As https://github.com/kubernetes/kubernetes/pull/100181#discussion_r594825941 Name the API Server-related master method  `APIServer` instead of the `ControlPlane`.
Rename master:
-  InitTestMaster -> InitTestAPIServer
- initTestMaster -> initTestAPIServer

#### Which issue(s) this PR fixes:

Part of #94900

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
LGTM

**Comment by sftim** [comment]
/retitle Rename master to apiserver for some test cases

**Comment by mengjiao-liu** [comment]
To more accurately describe the scope of the changes, I modified the COMMIT information.

**Comment by mengjiao-liu** [comment]
@sftim It looks like the upper case LGTM doesn't work.
for lgtm

**Comment by mengjiao-liu** [comment]
To more accurately describe the scope of the changes, I modified the COMMIT information.  Thank you for your review. @sftim

**Comment by mengjiao-liu** [comment]
ping @caseydavenport

File: test/integration/volumescheduling/util.go [hunk 1]
```diff
@@ -54,10 +54,10 @@ type testContext struct {
 	cancelFn context.CancelFunc
 }
 
-// initTestMaster initializes a test environment and creates a master with default
+// InitTestAPIServer initializes a test environment and creates a master with default
```
Comment (BenTheElder) [inline test/integration/volumescheduling/util.go:57]: making this method public seems unrelated to the goal of this PR

**Comment by BenTheElder** [comment]
currently CI is saying this code doesn't compile
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/102877/pull-kubernetes-typecheck/1405419794433314816
happy to review more when it does

File: test/integration/volumescheduling/util.go (hunk 1 above)
Comment (mengjiao-liu) [reply test/integration/volumescheduling/util.go:57]: I didn't see the case when I replaced it, so let me fix that.

**Comment by mengjiao-liu** [comment]
oh,a conflict `test/integration/scheduler/util.go:108` was missed while rebasing👀.  It has been updated now. Thanks for your review.@BenTheElder

File: test/integration/volumescheduling/util.go (hunk 1 above)
Comment (mengjiao-liu) [reply test/integration/volumescheduling/util.go]: Updated.

**Review by BenTheElder — APPROVED** [review]
/lgtm
/approve
Thanks!!
